03_method/01_method_first_step.rb

```diff
@@ -8,11 +8,31 @@
 # - 4. `F1.new.respond_to?(定義していないメソッド名)` を実行したときにtrueを返すこと
 
 class F1
+  define_method :hello do
+    'hello'
+  end
+
+  class << self
+    define_method :world do
+      'world'
+    end
+  end
```

Comment:
F1クラスのクラスメソッドとして定義する（特異メソッドを定義する）ので、`define_singleton_method`も使えるとのこと。
```suggestion
  define_singleton_method :world do
    'world'
  end
```